test/controllers/home_controller_test.rb

```diff
@@ -0,0 +1,9 @@
+require 'test_helper'
+
+class HomeControllerTest < ActionController::TestCase
+  test "should get home" do
```

Comment:
Also feel free to chat with any of us about why we like to test things the way we do.